pkg/controller/nodelifecycle/node_lifecycle_controller.go

```diff
@@ -728,6 +862,8 @@ func (nc *Controller) monitorNodeHealth() error {
 
 	zoneToNodeConditions := map[string][]*v1.NodeCondition{}
 	for i := range nodes {
+		continueToOuterLoop := false
```

Comment:
This is not what we generally do. You may use done or skip to break out.

Reply:
fixed for better readable code.